src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1619,7 +1773,49 @@ There are two configuration options to control this mechanism (present in the `l
 ** The pa.ldap.tenant.attr configuration is not defined.
 ** The pa.ldap.tenant.attr configuration is defined, but the configured attribute is empty for the user.
 
-=== PAM
+[[_ldap_with_multiple_domains]]
+=== LDAP authentication with Multiple Domains
+
+The ProActive server can be configured to authenticate users according to *multiple LDAP domains*.
+
+Following is a step-by-step procedure to configure it.
+
+1. Copy the file `PROACTIVE_HOME\config\authentication\ldap.cfg` into multiple files, one for each domain.
```

Comment:
Copy the file `PROACTIVE_HOME\config\authentication\ldap.cfg` **multiple times in the same directory, such as to have one file copy per domain.**